sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/communication/notification/Notification.groovy

```diff
@@ -0,0 +1,14 @@
+package life.qbic.portal.sampletracking.communication.notification
+
+/**
+ * <h1><short description></h1>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <versiontag>
+ */
+class Notification {
+    static enum Type {
+
+    }
```

Comment:
nothing in here?

Reply:
Yes, I committed it by accident. It is not relevant and I will remove it